README.md

```diff
@@ -82,6 +82,23 @@ You can also run each process individually from separate terminals with the foll
 
 We have a number of default users seeded for local development, the most basic of which being (email: `admin@example.com`, password: `abcdef`) which will login to both the public and partners sites, but you can view other default seeded users and their permissions by checking out the user section of the [seed file](https://github.com/bloom-housing/bloom/blob/aed77bf06525be359ef9205044fabbea2ab2576d/api/prisma/seed-staging.ts#L67).
 
+### Running locally in docker
+
+Run the docker images to emulate a cloud deployment locally on your machine. Docker and Docker
+Compose are required. The easiest way is to install [Docker
+Desktop](https://docs.docker.com/desktop/).
+
+The database is seeded with the `yarn:dbseed:development` script.
```

Comment:
suggestion: We typically test with the "staging" data as that one is more realistic. The "dev" data is more randomly generated which we mostly use to test large number of things or try to identify edge cases. "staging" data is more consistent and has all the different scenarios we need to account for.